### DISCUSSION TITLE
Gl-iNet Flint - No Speedify Option under "VPN" Tab on 0.99.9

### DISCUSSION THREAD

**Discussion by netninja71** [SmoothWAN/SmoothWAN#108] [General & Help]
This could totally be me doing something wrong, but I've flashed 0.99.9 to my Flint twice, and the Speedify option doesn't show up under "VPN." Am I missing something, or is there another way to install it?

**Comment by TalalMash** [comment]
Flint users are active at https://github.com/SmoothWAN/SmoothWAN/issues/69 , as I don't have the device, they may chime in.

**Reply by netninja71** [reply]
Headed there now, thank you!
